### DISCUSSION TITLE
Share.RequestAsync Feedback

### DISCUSSION THREAD

**Discussion by michelmoorlag** [CommunityToolkit/Maui#355] [New Feature Discussions]
Users of our app can invite other users using the share api. For our information it would be nice if we could see who invited who. For that reason it would be nice if the Share.RequestAsync could return some information like the app that was uses to share and with which contact(s) it was shared with. I don't know if this is possible but it would be very usefull for us and possibly others.

One other thing. We use a unique invite code for our app that can only be used by one user. It would be nice if we could limit the number of users you can share with to one person. I don't know if this is possible but it would be very usefull for us and possibly others.

**Comment by pictos** [comment]
@michelmoorlag thanks for the proposal, but we don't host the `Share` API it lives on .NET MAUI, that said can you open this proposal there?

**Reply by TheCodeTraveler** [reply]
Here's the link to the .NET MAUI Repo where the `Share` API lives: https://github.com/dotnet/maui

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
